### ISSUE TITLE
reset cache when new post

### ISSUE THREAD

**Issue by B4rb3rouss** [BlogoText/blogotext#362]
When a new post is published, a mecanism to reload cache might be considered. (for rss.php as example).

**Comment by B4rb3rouss** [comment]
Maybe when submitting the form when posting a new article, a small PHP line can set the cache time too old : 

```
$flux_cache_lv2_path = DIR_VHOST_CACHE.'cache2_'. $format .'_'.substr(md5((isset($_SERVER['QUERY_STRING'])) ? $_SERVER['QUERY_STRING'] : ''), 0, 8).'.dat';

// if cache file exists
if (is_file($flux_cache_lv2_path)) {
    touch($flux_cache_lv2_path);
}
```

**Comment by remrem** [comment]
Yep, considering that the feed must be kept up-to-date when adding / removing content, it seems to me to be a bug.

**Comment by remrem** [comment]
I took a look at the cache system (level 2) and there is no way to remove the caches related to articles, you must delete everything. So, I think the best is to push back to the 3.8 and review it a bit, at least the naming system for cache files.

Good for you ?

**Comment by B4rb3rouss** [comment]
As I have no better answer, yes, that's good to me.
